src/test/ActionsRouter.sol

```diff
@@ -100,8 +108,8 @@ contract ActionsRouter is IUnlockCallback, Test {
     }
 
     function _assertReservesEquals(bytes memory params) internal view {
-        (Currency currency, uint256 expectedReserves) = abi.decode(params, (Currency, uint256));
-        assertEq(manager.getReserves(currency), expectedReserves, "reserves value incorrect");
+        (, uint256 expectedReserves) = abi.decode(params, (Currency, uint256));
```

Comment:
why does it still decode a currency at all? shouldnt it just only pass in the uint256 now?

Reply:
does not need it, good catch!